### PR TITLE
Fix ReSpec issues

### LINKED ISSUE #77: Fix ReSpec warnings on GitHub spec
image

https://w3c.github.io/navigation-timing/

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #77

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/navigation-timing/pull/80.html" title="Last updated on Nov 29, 2017, 3:04 AM GMT">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/navigation-timing/afdcce9...6ebf3b2.html" title="Last updated on Nov 29, 2017, 3:04 AM GMT">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
This fixes all the ReSpec issues, but only partially fixes some of the problems in the spec with regards to markup.

**Comment by marcoscaceres** [comment]
I also took the liberty of fixing the main example... `console.table()` is much better there, and shows off `.toJSON()` too.

File: index.html [hunk 1]
```diff
@@ -1,1074 +1,1650 @@
...
-  Anne Van Kesteren,
-  Arvind Jain,
-  Boris Zbarsky,
-  Jason Weber,
-  Jonas Sicking,
-  James Simonsen,
-  Karen Anderson,
-  Nic Jansma,
-  Philippe Le Hegaret,
-  Steve So
```
Comment (siusin) [inline index.html]: This seems like an editorial bug, s/`href="https://heycam.github.io/!WEBIDL/#idl-enums"`/`data-cite="!WEBIDL/#idl-enums"` ?

Comment (siusin) [inline index.html]: it's in another section now, "`!HTML/semantics.html#attr-meta-http-equiv-refresh`"

Comment (siusin) [inline index.html]: -> "`!HTML/semantics.html#attr-meta-http-equiv-refresh`"

Comment (siusin) [inline index.html]: -> "`!HTML/semantics.html#attr-meta-http-equiv-refresh`"

**Comment by siusin** [comment]
@marcoscaceres Thanks for getting this done \o/

Everything looks good, except for a few broken links.

**Comment by marcoscaceres** [comment]
[Lots of broken links still](https://validator.w3.org/checklink?uri=https%3A%2F%2Fpr-preview.s3.amazonaws.com%2Fw3c%2Fnavigation-timing%2Fpull%2F80.html&hide_type=all&depth=&check=Check).

Should we do those as a new PR?

**Comment by marcoscaceres** [comment]
Did  couple more... that's unfortunately all the time I have to spend on this, as I'm working on other projects. However, all the broken links are easy to fix from here.... but there are a few things that don't exist in HTML anymore that need to be updated in this spec (e.g., "HTTP response codes equivalence").

**Comment by siusin** [comment]
Thanks @marcoscaceres ! I'll try to fix the other broken links, as well as the ReSpec issues in the RT spec, please leave them to me :)

**Comment by marcoscaceres** [comment]
Thanks @siusin! If you have any ReSpec questions just email me or ping me on IRC "marcosc". And, I'm always more than happy to review stuff.

**Pull request merged**